manage.py

```diff
@@ -108,6 +108,12 @@ def seed():
     # Scrape articles for feed
     scrape_articles(Categorizer())
 
+    # Add items to user_item relationship
```

Comment:
What do we need to debug with it?

-- 
Alex Gerstein
gerstein.alex@gmail.com

On February 20, 2015 at 2:40:12 PM, Scott Gladstone (notifications@github.com) wrote:

I think we should do it for one of the feeds so we have it for debugging purposes. I agree that it doesn't need to be done for all of them.